File: python/program/Clothing.py
from Product import Product

# deklarasi class

# deklarasi class menjadi anak dari product


class Clothing(Product):

    # deklarasi private atribut
    __size = ""
    __material = ""
    __gender = ""

    # constructor

    def __init__(self, idProduct="", nama="", brand="", price="", size="", material="", gender=""):
        super().__init__(idProduct, nama, brand, price)
        self.__size = size
        self.__material = material
        self.__gender = gender

    # setter dan getter
    def get_size(self):
        return self.__size

    def set_size(self, size):
        self.__size = size

    def get_material(self):
        return self.__material

    def set_material(self, material):
        self.__material = material

    def get_gender(self):
        return self.__gender

    def set_gender(self, gender):
        self.__gender = gender

File: php/program/Clothing.php
<?php

// import file
require ('Product.php');

// deklarasi kelas Clothing
class Clothing extends Product {
    // atribut private
    private $size = '';
    private $material = '';
    private $gender = '';

    // construktor
    public function __construct( $size = '', $material = '' , $gender = '') {
        $this->size = $size;
        $this->material = $material;
        $this->gender = $gender;
    }
    // getter dan setter 
    public function get_size(){
        return $this->size;
    }
    public function set_size($size){
        $this->size = $size;
    }

    public function get_material(){
        return $this->material;
    }
    public function set_material($material){
        $this->material = $material;
    }
    
    
    public function get_gender(){
        return $this->gender;
    }
    public function set_gender($gender){
        $this->gender = $gender;
    }
}

?>